sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/_models.py

```diff
@@ -513,104 +513,104 @@ def _from_certificate_operation_bundle(cls, certificate_operation_bundle):
 
     @property
     def id(self):
-        # type: () -> str
-        """:rtype: str"""
+        # type: () -> Optional[str]
+        """:rtype: optional[str]"""
         return self._id
 
     @property
     def name(self):
-        # type: () -> str
-        """:rtype: str"""
-        return self._vault_id.name
+        # type: () -> Optional[str]
+        """:rtype: optional[str]"""
+        return self._vault_id.name if self._vault_id else None
 
     @property
     def vault_url(self):
-        # type: () -> str
+        # type: () -> Optional[str]
         """URL of the vault containing the CertificateOperation
 
-        :rtype: str
+        :rtype: optional[str]
         """
-        return self._vault_id.vault_url
+        return self._vault_id.vault_url if self._vault_id else None
 
     @property
     def issuer_name(self):
-        # type: () -> str
+        # type: () -> Union[str, WellKnownIssuerNames, None]
         """The name of the issuer of the certificate.
 
-        :rtype: str
+        :rtype: union[str, WellKnownIssuerNames, None]
```

Comment:
Similarly to using `Type or None` for `Optional[Type]`, I would probably change this to
```suggestion
        :rtype: str or WellKnownIssuerNames or None
```